common/components/mlflow_model_local_validation/spec.yaml

```diff
@@ -20,6 +20,9 @@ command: |
   cp ${{inputs.model_path}}/conda.yaml ./
   conda env create -p $ENV_PREFIX -f conda.yaml -q
   echo "conda env successfully created at $ENV_PREFIX"
+  echo "Install appinsights additionally, for logging"
```

Comment:
python-sdk-v2 has appinsights . Is it still needed


name: python-sdk-v2
channels:
  - conda-forge
dependencies:
  - python=3.8
  - pip=21.2.4
  - pip:
    - azure-core=={{latest-pypi-version}}
    - azure-ai-ml=={{latest-pypi-version}}
    - azure-identity=={{latest-pypi-version}}
    - azureml-core=={{latest-pypi-version}}
    - applicationinsights=={{latest-pypi-version}}